src/components/CalendarView/DayTileItem.js

```diff
@@ -49,11 +49,18 @@ const DayTileItemGroup = ({ children }) =>
     return [<DayTileItem.Divider />, child];
   });
 
-const DayTileItemBaseText = styled(Text).attrs({ size: "uno" })`
+const DayTileItemBaseText = styled(Text).attrs(() => ({ size: "uno" }))`
   width: 100%;
   word-break: break-word;
 `;
 
+const DayTileItemLabel = styled(DayTileItemBaseText).attrs(() => ({
+  weight: "semiBold"
+}))`
+  text-transform: uppercase;
+  color: ${getLabelTextColor};
```

Comment:
Are you missing an argument and/or invocation here?

Reply:
No, `getLabelTextColor` takes `variant` from `props`. This is a short version of `${ ( {variant} ) => getLabelTextColor( {variant} ) }`.

Reply:
Oh ok, cool